pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -598,7 +598,9 @@ func (cp *capacityPlugin) buildHierarchicalQueueAttrs(ssn *framework.Session) bo
 
 	// init root queue realCapability/capability/deserved as cp.totalResource
 	rootQueueAttr := cp.queueOpts[api.QueueID(cp.rootQueue)]
-	rootQueueAttr.capability = cp.totalResource
+	if rootQueueAttr.capability == nil || rootQueueAttr.capability.IsEmpty() {
```

Comment:
I think `rootQueueAttr.capability.IsEmpty()` is enough because the capability of root queue will be initialized in `newQueueAttr`, and it won't be nil